src/main/res/values/strings.xml

```diff
@@ -640,13 +640,13 @@
     <string name="welcome_chat_over_email">Secure Decentralized Chat</string>
     <string name="scan_invitation_code">Scan Invitation Code</string>
     <string name="login_title">Log In</string>
-    <string name="login_advanced_hint">This login is for advanced users:\n\n• Do not use an account you\'re using in another app.\n\n• Classic e-mail server allow chats without end-to-end encryption marked by a mail icon.</string>
+    <string name="login_advanced_hint">This login is for advanced users:\n\n• Do not use an account you\'re using in another app.\n\n• Classic email server allow chats without end-to-end encryption marked by a mail icon.</string>
```

Comment:
While we are changing it, shouldn't "mail icon" be called "envelope icon"?

Reply:
But we can also put emoji here, like "✉️ icon".

Reply:
i just tried - depending on the android version, the emoji is yellow or white, but does not really look like the actual thing.

and showing the real thing is some effort.

as it is advanced, it seems better to just show nothing here